sql-aggregates/on-the-map.sql

```diff
@@ -0,0 +1,4 @@
+select count("cities"."name") as "numberOfCities", "countries"."name"
```

Comment:
`count` actually ignores its argument, so it's customary to always use `count(*)`.